### DISCUSSION TITLE
How to get all the Edges in the cluster when you know id of one vertex in the group of nodes

### DISCUSSION THREAD

**Discussion by hotcoder** [JanusGraph/janusgraph#2570] [Q&A]
Node 'AAA' connected to other three nodes by edge type 'X'

B node connected to D by edge type  K
C node connected to D by edge type  K

And we will receive one input called ID of the AAA node, so I want all the  Edge nodes of type K, am not able to frame the gremlin query, can anyone please help me with that.

Given link contains an example for the above query with images, please help me.

https://lucid.app/lucidspark/a67c9729-50e7-428f-abe9-52b253b621d9/view#

**At least for example if you know the **id of the C node**, is it possible to get **all identity edges** in this cluster of nodes?**

**Comment by li-boxuan** [comment]
What do you mean by "Edge nodes" and "identity edges"?

**Comment by storyteller28th** [comment]
why cannot you use the gremlin query?
